frontend/apps/filemanager/filemanager.lua

```diff
@@ -460,6 +460,7 @@ function FileManager:init()
         ui = self
     }
     self.active_widgets = { Screenshoter:new{ prefix = 'FileManager' } }
+    table.insert(self, self.active_widgets[1])
```

Comment:
May be to have that clearer:
```lua
local screenshoter = Screenshoter:new{ prefix = 'FileManager' }
self.active_widgets = { screenshoter } -- so that xxx
table.insert(self, screenshoter) -- so that yyy
```
(and same comments in readerui so we don't wonder why when we rereader that in some years :)